lib/onfido/resources/address.rb

```diff
@@ -1,10 +1,7 @@
 module Onfido
   class Address < Resource
     def all(postcode)
-      get(
-        path: 'addresses/pick',
-        payload: { postcode: postcode.delete(' ') }
-      )
+      get(path: "addresses/pick?postcode=#{postcode.delete(' ')}")
```

Comment:
I think `RestClient` which is used internally supports a `:params` option that would have to be added here but would generally be a cleaner way to pass the query param:
https://github.com/onfido/onfido-ruby/blob/master/lib/onfido/resource.rb#L22-L36

Reply:
👍 I ticketed this up to do as a separate change, as discussed on slack.